vispy/testing/_runners.py

```diff
@@ -111,6 +112,30 @@ def _unit(mode, extra_arg_string, coverage=False):
         os.rename(op.join(cwd, '.coverage'), out_name)
 
 
+def _docs():
+    """test docstring paramters
+    using vispy/utils/tests/test_docstring_parameters.py"""
+    dev = _get_import_dir()[1]
+
+    if not dev:
+        warnings.warn("Docstring test imports Vispy from"
+                      " Vispy's installation. It is"
+                      " recommended to setup Vispy using"
+                      " 'python setup.py develop'"
+                      " so that the latest sources are used automatically")
+    try:
+        # this should always be importable
+        from vispy.util.tests import test_docstring_parameters
+        print("Running docstring test...")
+        test_docstring_parameters.test_docstring_parameters()
```

Comment:
It is best/clearest to put as few lines as possible in the `try`. So that would be the actual function call. Also, you can use a relative import here like `from ..util.tests import test_docstring_parameters`.